R/create-individual.R

```diff
@@ -0,0 +1,87 @@
+#' Creates an individual using the PKSim Database.
+#'
+#' @param path A string representation of a path, with path entries separated
+#'
+#' @return An array of \code{ParameterValue} containing the value of each individual parameter
+#' @examples
+#'
+#' @export
+createIndividual <- function(originData, useDistribution = FALSE, moleculeOntogenies = NULL) {
+  validateIsOfType(originData, OriginData)
+  validateIsLogical(useDistribution)
+  validateIsOfType(moleculeOntogenies, MoleculeOntogeny, nullAllowed = TRUE)
+  individualFactory <- rClr::clrCallStatic("PKSim.R.Api", "GetIndividualFactory")
+  moleculeOntogenies <- c(moleculeOntogenies)
+
+  individualCharacteristics <-  rClr::clrNew("PKSim.R.Domain.IndividualCharacteristics")
+  rClr::clrSet(individualCharacteristics, "OriginData", originData$ref);
+  for (moleculeOntogeny in moleculeOntogenies) {
+    rClr::clrCall(individualCharacteristics, "AddMoleculeOntogeny", moleculeOntogeny$ref);
+  }
+
+  methodToCall <- if (useDistribution) "DistributionsFor" else "CreateIndividual"
+  netParams <- rClr::clrCall(individualFactory, methodToCall, individualCharacteristics)
+
+  outputList <- list(
+    paths = getPropertyValues(netParams, "ParameterPath"),
+    values = getPropertyValues(netParams, "Value")
+  )
+
+  if (useDistribution) {
+    distributionList <- list(
+      means = getPropertyValues(netParams, "Mean"),
+      stds = getPropertyValues(netParams, "Std"),
+      distributionTypes = getPropertyValues(getPropertyValues(netParams, "DistributionType"), "DisplayName")
+    )
+    outputList <- c(outputList, distributionList)
+  }
+
+  return(outputList)
+}
+
+getPropertyValues <- function(netObjects, propertyName) {
+  sapply(netObjects, function(x) rClr::clrGet(x, name = propertyName))
+}
+
+createOriginData <- function(

```

Comment:
A helper function that is going to be used to createOriginData 

Note that I am not validating that the species, population etc are in the predefined enum as it would prevent the user to work with user defined species in the PKSim DB. This could easily happen, especially during model validation